api/src/main/java/io/strimzi/api/kafka/model/KafkaMirrorMakerSpec.java

```diff
@@ -60,7 +63,7 @@
     private Map<String, Object> additionalProperties = new HashMap<>(0);
 
     @Description("The number of pods in the `Deployment`.")
-    @Minimum(1)
+    @Minimum(0)
```

Comment:
When you change something in the `api` module, it is not enough to just build the images. You have to at the end run `make crd_install` to make sure the new CRD definitions are copied to all the different places. This is why right now the CI builds are failing.

Reply:
I have executed `make crd_install`, but seems like nothing has been changed. (nothing to commit)
And in the `045-Crd-kafkamirrormaker.yaml` I still can see:
```
            replicas:
              type: integer
              minimum: 1
              description: The number of pods in the `Deployment`.
```

Reply:
So, you will first need to do `make all` (or you can go to the `api` module and do `mavn -DskipTests install`) -> that should generate the changes tot he CRD in `install/cluster-operator`. And once you have that you can do `make crd_install_`.